### PR TITLE
Add healthcheck

### PR DESCRIPTION
**Pull request by vasilvestre**
I asked dunglas if I could create another PR as he has no time actually to rebase his work in #391

I've added health check on PHP, most of his work already had been fixed and quay.io isn't used anymore for cache.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -93,6 +100,7 @@ COPY translations translations/
 RUN set -eux; \
     mkdir -p var/cache var/log; \
     composer dump-autoload --classmap-authoritative; \
+    composer dump-env prod; \
```
Comment (toofff) [inline Dockerfile:103]: I will remove the `.env` files from the image after this command.

**Review by toofff — APPROVED** [review]

**Comment by vasilvestre** [comment]
Any news to integrate it for 1.12 ?

**Comment by diimpp** [comment]
Hi , @vasilvestre, can you please elaborate why this change is necessary and what kind issue it's solving. Is it dev/prod functionality?

**Comment by vasilvestre** [comment]
> Hi , @vasilvestre, can you please elaborate why this change is necessary and what kind issue it's solving. Is it dev/prod functionality?

It helps to know if your container is heathy

**Pull request closed without merging**